.github/workflows/lint.yml

```diff
@@ -0,0 +1,16 @@
+name: Lint
+
+on: [push]
+
+jobs:
+  eslint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Install modules
+        run: yarn
+        working-directory: ./js
```

Comment:
There's a [convenient way](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#defaultsrun) to apply it globally to the whole workflow.

```
defaults:
  run:
    working-directory: js
```

Reply:
Ok

Reply:
Sorry, don't see any changes yet